LocalPackages/DataBrokerProtection/Tests/DataBrokerProtectionTests/DataBrokerProfileQueryOperationManagerTests.swift

```diff
@@ -369,24 +369,17 @@ final class DataBrokerProfileQueryOperationManagerTests: XCTestCase {
         }
     }
 
-    func testWhenRemovedProfileIsFound_thenOptOutConfirmedIsAddedRemoveDateIsUpdatedAndPreferredRunDateIsSetToNil() async {
+    func testWhenRemovedProfileIsFound_thenOptOutConfirmedIsAddedRemoveDateIsUpdated() async {
         do {
-            let extractedProfileId: Int64 = 1
-            let brokerId: Int64 = 1
-            let profileQueryId: Int64 = 1
-            let mockHistoryEvent = HistoryEvent(extractedProfileId: extractedProfileId, brokerId: brokerId, profileQueryId: profileQueryId, type: .optOutRequested)
-            let mockBrokerProfileQuery = BrokerProfileQueryData(
-                dataBroker: .mock,
-                profileQuery: .mock,
-                scanJobData: .mock,
-                optOutJobData: [.mock(with: .mockWithoutRemovedDate, preferredRunDate: Date(), historyEvents: [mockHistoryEvent])]
-            )
-
```

Comment:
This was incorrectly commited and I'm reverting it back to the previous version.